### PR TITLE
Add Cookie Deserialization Logging

### PR DESCRIPTION
**Pull request by DaveLak**
Adds logging to handle errors when trying to parse cookie JSON. The `xdomain-cookies` library already swallows it's errors in a try catch so only our `JSON.parse` errors need to be handled.

### CODE REVIEW

File: src/attribution.js [hunk 1]
```diff
@@ -48,18 +49,18 @@ export function attribute(attributionDoneCallback) {
   }
 
   const utmCookieJar = loadAndSaveCookieJar();
-  utmCookieJar.get('utmFirst', utmFirst => {
+  utmCookieJar.get(COOKIE_LABELS.UTM_FIRST, utmFirst => {
     if (!utmFirst) {
       utmCookieJar.set(
-        'utmFirst',
+        COOKIE_LABELS.UTM_FIRST,
```
Comment (tday) [inline src/attribution.js]: We don't want utmFirst and utmLast to be configurable. They shoudl be a const somewhere

**Comment by DaveLak** [comment]
[utm-attribution can't be parsed](https://trello.com/c/WpTPmyqX/1010-utm-attribution-cant-be-parsed)

File: src/attribution.js [hunk 2]
```diff
@@ -90,6 +91,24 @@ function getCookieSafeQueryString(unsafeQueryString) {
...
+      severity: 'error',
+      context: 'attribution cookie value deserialization',
+      metaData: {
+        error,
+        cookieName,
+        cookieValue: maybeInvalidJSON,
+      },
+    };
+    tryReportError(new Error('Failed to parse cookie JSON'), errorOpts);
+    return false;
```
Comment (tday) [inline src/attribution.js]: Just return empty string to avoid mixed types? They made it falsy for a reason...

File: src/errorReporter.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+export function tryReportError(err, opts) {
+  const isBudgingAvailable = window.bugsnagClient && window.bugsnagClient.notify;
```
Comment (tday) [inline src/errorReporter.js]: "Budging"?

File: src/attribution.js (hunk 2 above)
Comment (DaveLak) [reply src/attribution.js]: > Just return empty string to avoid mixed types? 
Yeah, fair.

> They made it falsy for a reason...
😐

Comment (DaveLak) [reply src/attribution.js]: This actually doesn't return a string, rather an object (via `JSON.pare()`) so I've updated this to return an empty object.

**Pull request merged**